### ISSUE TITLE
ossec.conf changes not persisting after container restart

### ISSUE THREAD

**Issue by Geronaso** [wazuh/wazuh-docker#656]
Hello, I have built the multi-node configuration.

Whenever I restart the container the ossec.conf at /var/etc/etc/ is written back and the configurations does not persist.

Is there any work around for this, or is this the expected behavior?

Thanks for the help

**Comment by rauldpm** [comment]
Hello @Geronaso 

Could you tell us what version of wazuh are you deploying? how are you trying to make the changes?

Are you changing the ossec.conf path? The path of this file is `/var/ossec/etc` and not `/var/etc/etc`

If you are deploying 4.3.x, you can modify these files:

- Multi node:
  - https://github.com/wazuh/wazuh-docker/blob/4.3/multi-node/config/wazuh_cluster/wazuh_manager.conf
  - https://github.com/wazuh/wazuh-docker/blob/4.3/multi-node/config/wazuh_cluster/wazuh_worker.conf

The change made to this file should be reflected inside the container every time it starts

**Comment by Geronaso** [comment]
Thanks for the reply @rauldpm

I am deploying the version 4.3.1.

I am sorry, I typed wrong. The correct path is /var/ossec/etc. I tried two approaches, one was to change through the Dashboard and restarting the Wazuh manager through the dashboard. And the other was to log in the master node container and change the file in /var/ossec/etc. However neither worked.

I tried your approach, and restarted all containers. However, now I am getting a strange error in the API Connection with the worker node.

![image](https://user-images.githubusercontent.com/17254859/171047539-7170bd5e-bdac-4c77-afab-965ca56bb5f7.png)
